CHANGELOG.md

```diff
@@ -1,13 +1,53 @@
+  - Fixed an update error when trying to bulk-update a plugin which is not installed. Thanks [Gary Jones] for reporting. [#442], [#508]
+  - Fixed admin notices display class. Props [Ninos Ego] and [Primoz Cigler]. [#478], [#495], [#509]
+  - Fixed an issue resulting in notices about installed/updated plugins on the bulk install/update pages being displayed at the top of the page instead of inline. [#510], [#511]
+
+* **Enhancements**:
+  - The full admin notice is now only displayed to users who can install/update/activate plugins. A limited _"Contact the site admin."_ notice is shown to select users if it pertains to _required_ plugins. The selection of which users get to see this last message is based on the `publish_posts` (=Author) capability. This capability is however filterable using the new `tgmpa_show_admin_notice_capability` filter. Thanks [Stanislav Khromov], [Gary Jones], [Mickey Kay], [Ollie Treend] for suggesting. [#190], [#414], [#489], [#507]
+  - The example file now shows examples of different ways for including TGMPA based on the context in which you are using it. Props [Emil Uzelac] for the suggestion. [#469], [#503]
+
+* **I18N improvements**:
+  - Improved some text strings and translator messages. Props [Rami]. [#516]
+  - Added translator messages for all strings with variable replacement.[#563]
+  - Added `load_textdomain()` calls. [#521]
+  - Added translations for Brazilian Portuguese, Croatian, Czech, Dutch, French, German, Russian and Swedish [#450], [#574], [#570], [#465], [#524], [#528], [#543], [#561] with grateful thanks to [Elvis Henrique Pereira], [Denis Žoljom], [Karolína Vyskočilová], [geoclaps], [Hedi Chaibi], [Marciel Bartzik], [Vladislav Burlak] and [Lewis Porter].
+    Additionally translations for Australian English, Canadian English, British English, Esperanto, Spanish, Hebrew, Italian, Romanian and Serbian were added based on existing translations available in GlotPress. [#564]
+
+    Altogether, this means that the first TGMPA version being shipped with translation files covers 17 languages, which is awesome !
+    More translations are of course welcome, so please send the .po file(s) in as a pull request.
+
+    _Please note: If you download TGMPA using the custom generator and indicate it's for a theme to be hosted on wordpress.org, you will receive a version without the `load_textdomain()` calls or the translation files.
+    Theme check rules dictate that you should only use one textdomain in your theme and the localization calls will be adjusted to use your theme's textdomain.
+	As most TGMPA strings have a lot of translations available in GlotPress already, this should not cause any real issues._
+
+* **Housekeeping**:
+  - Various other minor improvements and keeping things in line with WP core. [#512], [#513], [#514], [#532] (thanks [Utkarsh Patel]), [#562], [#571]
```

Comment:
That's weird ... I put a space between the `]` and `(` to prevent that. Never mind, will use a dash instead.